awx/ui_next/src/screens/Template/shared/JobTemplateForm.jsx

```diff
@@ -37,11 +40,14 @@ import {
   InstanceGroupsLookup,
   ProjectLookup,
   MultiCredentialsLookup,
+  CredentialLookup,
 } from '@components/Lookup';
-import { JobTemplatesAPI, ProjectsAPI } from '@api';
+import { JobTemplatesAPI, ProjectsAPI, CredentialTypesAPI } from '@api';
 import LabelSelect from './LabelSelect';
 import PlaybookSelect from './PlaybookSelect';
 
+const urlOrigin = window.location.origin;
```

Comment:
I noticed we were using document.origin for the callback stuff in here...I don't care if we use that or this, but if we can make things consistent rather than introducing another pattern, I think that'd be ideal.

Reply:
good catch.  I use the document.origin but I'll have to move it outside of the function.